### DISCUSSION TITLE
Question about text

### DISCUSSION THREAD

**Discussion by jay-wl** [SVG-Edit/svgedit#524] [Q&A]
How Can I rasterize a text field to a path 
Also, how can I change the width of a text box (basically skewing the text to fit in a preset box )

**Comment by jfhenon** [comment]
This is not a feature currently available. We are in the process to refresh svg-edit that was not actively maintained for a while so it is unlikely we add anything like this in the short term. 
However, if you really need it, you may build a new extension (several examples in the SVG-edit code).

**Comment by jay-wl** [comment]
Thankyou jfhenon,
I have one other function I am working on. I want to be able to add text programmatical to a text box while the text box retains its height and width attributes. Basically skewing the text to fit in the textbox so that the new word can fit in the same spacing as the old word.

also, 
I actually am working on an extension to rasterize the text and I would be glad to post it when it is completed

**Comment by ile** [comment]
Hi @jfhenon,

Can I ask what are your plans in the future for this? How will you refresh this?

There are some features I might need, so I may get into developing this on my own (I can publish those changes if I make any). But to avoid any duplicate work, I'm interested what you are going to do?

Thanks.

**Comment by jfhenon** [comment]
Hello @ile, we recently merged with @brettz9 a first step towards a better organisation of the project, the next step is to review/simplify build tools, refresh external librairies (using npm packages). 
We will have also to review how we can split a few files such as svgedit.js, svgcanvas.js into smaller pieces to make them more manageable. I also would like to improve the css to make svgedit look a little more modern.
In parallel, we will also have to fix the most critical bugs...
This means that it will be difficult to add new features before the end of the year. It would just be excellent news if you want to contribute to the project.

**Comment by ile** [comment]
Hey,

Ok. I will do the features I need with the current codebase (and into my own fork) and when you are done, we can see if you can use those. I only took a quick look at the code and everything looked nice already, so I think this plan will work fine. 

So, let's see where this will take ... I'll get back when/if I have something to publish.
